lib/client.js

```diff
@@ -1,46 +1,43 @@
-(function() { // begin enclosed
+(function () {
+  // begin enclosed
```

Comment:
Are these changes done automatically in the IDE? My concern here is that there are a lot of syntax changes happening, that do not affect the logic, the actual updates are getting obscured by this, making it hard to review important changes

Reply:
will revert and redo changes, looks like IDE is restructuring